README.md

```diff
@@ -60,22 +60,36 @@ Provides a JupyterHub Service on an existing Openstack Cluster. This uses the he
 - Some metrics can't be selected by node name in Grafana dashboard as it requires a reverse DNS.
 
 ## Requirements
+The following assumes you have an Ubuntu 20.04 machine with `pip3`, `python3` and `python3-venv` already installed.
 
-- Ansible ([Installing Ansible — Ansible Documentation](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html))
+
+- Ansible ([Installing Ansible — Ansible Documentation](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html)) - this is installed using `pip` in `py-requirements.txt`
 - Helm 3 ([Installing Helm](https://helm.sh/docs/intro/install/))
 - kubectl ([Install Tools | Kubernetes](https://kubernetes.io/docs/tasks/tools/))
-- Python 3
-  - `ansible, kubernetes`
+
+Helm and kubectl can be installed using snap:
+
+```bash
+sudo apt-get update && sudo apt-get install -y snapd
+export PATH=$PATH:/snap/bin
+sudo snap install kubectl --classic
+sudo snap install helm --classic
+```
+
 ### Local Environment Setup
-- Upgrade pip3 as the default version is too old to handle the required deps: `pip3 install pip --upgrade`
-- Activate .venv if present then install pip deps: `pip3 install ansible setuptools setuptools-rust pyyaml kubernetes`
-- Clone the repository and cd into it
-- Install requirements `ansible-galaxy collection install -r requirements.yml`
+
+- Upgrade `pip3` using `pip3 install pip --upgrade` to ensure you are using the latest version of pip.
```

Comment:
We should do this after the venv step, else you'll potentially trash the system pip